apps/wallet/src/utils/transaction/swap/jupiter.ts

```diff
@@ -0,0 +1,91 @@
+import { VersionedTransaction } from '@solana/web3.js';
+import { logger } from '@walless/core';
+import type { TokenDocument } from '@walless/store';
+import { environment } from 'utils/config';
+
+import type { SwapQuote } from './types';
+
+export const getAliasedMint = (token: TokenDocument) => {
```

Comment:
Does this function convert a `devnet` mint address to a `mainnet-beta` mint address? I don't get the function name but inferring from its implementation

Reply:
No, this function primarily maps so; to wrapped sol which is used to handle as sol in `solana program`. I also add a dev case to easily test the flow (because swap API endpoint does not have any dev tool'

Reply:
You may add some comments to brief this purpose in case someone else may work with it in the future